### ISSUE TITLE
How to update mean and variance?

### ISSUE THREAD

**Issue by xmoanvaf** [WeitaoVan/L-GM-loss#7]
Hi, I'm currently using lgm loss in my research project, i'm wondering how to update mean and variance under the setting?

**Comment by WeitaoVan** [comment]
Hi, they are updated by standard gradient descent, with a proper initial value and a relatively low learning rate compared to the overall learning rate.
